travis/cfg/travis_run_pylint_pr.cfg

```diff
@@ -63,5 +67,4 @@ ignore-docstrings=yes
 notes=
 
 [IMPORTS]
-deprecated-modules=pdb,pudb,ipdb
-
+deprecated-modules=openerp.osv
```

Comment:
Pdb is checked by pylint stable config file.
"Deprecated module" is disabled in pr config file before of this change.
El sep 23, 2015 1:47 AM, "Pedro M. Baeza" <notifications@github.com>
escribió:

> In travis/cfg/travis_run_pylint_pr.cfg
> <https://github.com/OCA/maintainer-quality-tools/pull/243#discussion_r40174715>
> :
>
> > @@ -63,5 +67,4 @@ ignore-docstrings=yes
> >  notes=
> >
> >  [IMPORTS]
> > -deprecated-modules=pdb,pudb,ipdb
> > -
> > +deprecated-modules=openerp.osv
>
> Are now pdb and company allowed?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OCA/maintainer-quality-tools/pull/243/files#r40174715>
> .
>